helm/hpcc/values.schema.json

```diff
@@ -1007,6 +1007,11 @@
           "description": "The default storage plane to write data files to",
           "type": "string"
         },
+        "directAccessPlanes": {
```

Comment:
I think this might be what the "storagePlanes" property was intended for - I don't think it is currently used.  Worth removing it (or using it).

Reply:
Removed "storagePlanes".  I think directAccessPlanes is more descriptive of how they are actually used.